app/api/routers/breeze_buddy/integrations/handlers.py

```
@@ -0,0 +1,127 @@
+        role=UserRole.ADMIN,
+        reseller_ids=["*"],
+        merchant_ids=["*"],
+    )
+
+    try:
+        result = await push_lead_handler(lead_req, system_user)
+        logger.info(
+            f"woocommerce: order {order_id} queued as lead "
+            f"({result.get('lead_call_tracker_id')})"
+        )
+    except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)